mgmt_cli_test.py

```diff
@@ -320,17 +320,17 @@ def test_manager_sanity(self):
         4) test_client_encryption
         :return:
         """
-        with self.subTest('STEP 1: Basic Backup Test'):
+        with self.subTest('Basic Backup Test'):
             self.test_basic_backup()
-        with self.subTest('STEP 2: Repair Multiple Keyspace Types'):
+        with self.subTest('Repair Multiple Keyspace Types'):
             self.test_repair_multiple_keyspace_types()
-        with self.subTest('STEP 3: Mgmt Cluster CRUD'):
+        with self.subTest('Mgmt Cluster CRUD'):
             self.test_mgmt_cluster_crud()
-        with self.subTest('STEP 4: Mgmt cluster Health Check'):
+        with self.subTest('Mgmt cluster Health Check'):
             self.test_mgmt_cluster_healthcheck()
-        with self.subTest('STEP 5: Client Encryption'):
-            self.test_client_encryption()
         self.test_suspend_and_resume()
+        with self.subTest('Client Encryption'):
```

Comment:
maybe leave a comment here that this one should be the last ?

Reply:
Added!